<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMapElementClass</class>
 <widget class="QWidget" name="AddMapElementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddMapElement</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #2a2a2a;
	border: 3px solid #3b3e41;
	border-radius: 3px;
}

QLabel {
	border: 0px;
	color: white;
}

QPushButton#btnAccept, QPushButton#btnCancel {
    background-color: #3b3e41;
    border-radius: 3px;
    border: 1px solid #757575;
    font: bold 12px;
	color: white;
    min-width: 6em;
    padding: 6px;
}

QPushButton#btnAccept:hover, QPushButton#btnCancel:hover {
    background-color: #85b448;
	cursor: pointer;
}

QPushButton#btnAccept:pressed, QPushButton#btnCancel:pressed {
    background-color: #d86a39;
}

QDoubleSpinBox, QLineEdit {
    padding-right: 15px; /* make room for the arrows */
	background-color: #3b3e41;
    border: 1px solid #757575;
	border-radius: 0px;
	padding: 1px 3px;
	color: white;
}

QDoubleSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right; /* position at the top right corner */
    width: 16px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    border: 0px;
}

QDoubleSpinBox::up-arrow {
    image: url(Resources/arrow_up.png);
    width: 12px;
    height: 12px;
	padding-top: 3px;
	padding-right: 3px;
}

QDoubleSpinBox::up-arrow:hover {
	image: url(Resources/arrow_up_hover.png);
}

QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right; /* position at bottom right corner */
    width: 16px;
    border: 0px;
    border-top-width: 0;
}

QDoubleSpinBox::down-arrow {
    image: url(Resources/arrow_down.png);
    width: 12px;
    height: 12px;
	padding-bottom: 3px;
	padding-right: 3px;
}

QDoubleSpinBox::down-arrow:hover {
	image: url(Resources/arrow_down_hover.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="windowTitle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Add map element</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame_2">
       <property name="styleSheet">
        <string notr="true">QFrame {
	margin-top: 5px;
	border: 0px;
	padding: 0px;
margin-left: -5px;
}

QPushButton {
	border: 3px solid #3b3e41;
	border-radius: 3px;
}

QPushButton:hover {
	border: 3px solid #85b448;
}

QPushButton:pressed {
	border: 3px solid #d86a39;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="addMapElementDescription">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0px; margin-bottom: 3px;</string>
          </property>
          <property name="text">
           <string>Select decoration type:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mapElementsContainer">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="formFrame">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>New element name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Rotation:  </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAccept">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WarningErrorMsg.qrc"/>
 </resources>
 <connections/>
</ui>
